### ISSUE TITLE
USERS: what to do?

### ISSUE THREAD

**Issue by marl0rd** [marlontrujillo1080/eru#52]
There is a list of users in the Project Model. The main idea is to have authorization to see things in the display... What more can be done?

**Comment by gilbertojrequena** [comment]
It makes sense to have users when there is a centralized database, where multiple users can change the data. In our case we have a embedded local database which is supposed to be used just by one user. I don't see the benefit of having users using a local database

**Comment by marl0rd** [comment]
Yes. You are right with using database, but actually this entity is to log into the SCADA. For example:
 - Operators: When an operator enter in the SCADA using the username and password the button to increase the speed of a motor is disabled.
 - Engineers: When an engineer enter in the SCADA using the username and password the button to increase the speed is enabled.

Of course, this is handled by a welcome dialog in the SCADA to be logged. This a future feature really...

**Comment by gilbertojrequena** [comment]
It looks more like a role than an user

**Comment by marl0rd** [comment]
Yes it does..
